bears/python/requirements/PySafetyBear.py

```diff
@@ -49,7 +59,7 @@ def run(self, filename, file):
             return
 
         for vulnerability in safety.check(packages=packages):
-            if vulnerability.is_cve:
+            if cveKeyChecker([vulnerability]):
```

Comment:
why pass an array containing `vulnerability` ?  why not just pass `vulnerability`?

Reply:
No but list will effect performance

Reply:
sorry; please explain that?

Reply:
I am saying I uses the list during testing locally so, I passes same here but without parsing to list it will work even better